pages/settings/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button, Container, Input, Select, Stack } from '@chakra-ui/react';
+import { PrivilegeLevel } from '@prisma/client';
+import axios from 'axios';
+import { signIn, signOut, useSession } from 'next-auth/react';
+import { Dispatch, SetStateAction, useState } from 'react';
+import { BASE_URL } from '../../lib/constants';
+import {
+  getPrivilegeLevelFromSession,
+  privilegeLevelCompareTo,
+} from '../../lib/utils/privilegeLevelUtils';
+import { Alert, AlertIcon, AlertTitle, AlertDescription, CloseButton } from '@chakra-ui/react';
+
+const Settings: React.FC = () => {
+  const { data: session } = useSession();
+
+  const userPrivilegeLevel: PrivilegeLevel | undefined = getPrivilegeLevelFromSession(session);
+
+  if (session) {
+    return (
+      <>
+        Signed in as {session?.user?.email} <br />
+        {userPrivilegeLevel &&
+          privilegeLevelCompareTo(userPrivilegeLevel, PrivilegeLevel.ADMIN) >= 0 && (
+            <UpdatePrivilegeForm />
+          )}
+        <button onClick={() => signOut()}>Sign out</button>
+      </>
+    );
+  } else {
+    return (
+      <>
+        Not signed in <br />
+        <button onClick={() => signIn()}>Sign in</button>
+      </>
+    );
+  }
+};
+
+async function handleSubmit(
```

Comment:
This should call a method from an api client. So add an API client object (vsc/lib/apiClient.ts) and add the call to the privilege level route with the proper DTOs. In this page, we should use the API client object to submit the request, so basically calling the method like apiClient.updatePrivilegeLevel()